data/strings/strings.txt

```diff
@@ -5149,49 +5149,136 @@
     zh-Hant = 在螢幕上顯示
 
   [pref_map_style_title]
-    comment = Settings «Map» category: «Night style» title
-    tags = android,ios
-    en = Night Mode
-    af = Nagmodus
-    ar = الوضع الليلي
-    az = Gecə rejimi
-    be = Начны рэжым
-    bg = Нощен режим
-    ca = Mode nocturn
-    cs = Noční režim
-    da = Nattilstand
-    de = Nachtmodus
-    el = Νυχτερινή λειτουργία
-    es = Modo nocturno
-    et = Öörežiim
-    eu = Gaueko modua
-    fa = حالت شب
-    fi = Yötila
-    fr = Mode nuit
-    he = מצב לילה
-    hi = रात का मोड
-    hu = Éjszakai üzemmód
-    id = Mode Malam
-    it = Modalità notturna
-    ja = 夜間モード
-    ko = 나이트 모드
-    lt = Nakties režimas
-    mr = रात्र मोड
-    nb = Nattmodus
-    nl = Nachtmodus
-    pl = Tryb nocny
-    pt = Modo noturno
-    pt-BR = Modo noturno
-    ro = Mod nocturn
-    ru = Ночной режим
-    sk = Nočný režim
-    sv = Nattläge
-    th = โหมดกลางคืน
-    tr = Gece Modu
-    uk = Нічний режим
-    vi = Chế độ ban đêm
-    zh-Hans = 夜间模式
-    zh-Hant = 夜間模式
+    comment = Settings «Map» category: «Appearance» title
+    tags = android,ios
+    en = Appearance
+    ar = المظهر
+    az = Görünüş
+    be = Выгляд
+    bg = Изглед
+    ca = Aspecte
+    cs = Vzhled
+    da = Udseende
+    de = Erscheinungsbild
+    el = Εμφάνιση
+    es = Aspecto
+    et = Välimus
+    eu = Itxura
+    fa = ظاهری
+    fi = Ulkoasu
+    fr = Apparence
+    he = צבעים
```

Comment:
Since the comment by @biodranik mentioned it needs to equal iOS settings settings, my previous suggestion was bad. It should be:

he = מראה

(I had to look for a video in Hebrew showing these settings being changed to find it)

The other 2 are fine.